dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/CustomAccessLogFieldsTest.java

```diff
@@ -647,8 +647,8 @@ public boolean verify(String arg0, SSLSession arg1) {
             assertTrue("Nothing was returned from the servlet - there was a problem connecting.", lines.length() > 0);
             con.disconnect();
         } catch (IOException e) {
-            e.printStackTrace();
-            fail("Exception caught. Please see System.err log to view stack trace.");
+            //We can still log access log, just the status may return 500 in open liberty image
```

Comment:
Are we sure that all test code has waited for the server config change was complete before hitting the endpoint?  I am not comfortable to quietly moving forward when there is an exception.

Reply:
Yea I looked at the logs and the server config is updated before we hit the endpoint. I see it look for `CWWKG0017I: The server configuration was successfully updated in 0.072 seconds` and then we hit the endpoint. I think this must be only in the open liberty image builds that it returns 500. Another test defect also seemed to have this issue when hitting the landing page: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.workitem.viewWorkItem&id=276738
Anyways, this doesn't effect the access log logging. And it seems like the REST endpoints work fine `/metrics`. If the access log isn't logged, our test will catch it in the end.